kitchen/README.md

```diff
@@ -28,3 +28,26 @@ $ curl -v -XPOST -H "x-kitchen-cookies: a=b,c=d" $(hostname):PORT
 < Set-Cookie: c=d
 ...
 ```
+
+# Demos
+
+## Image Search and Tagging
+
+The demo application performs image search and then tags the images using the inception-v3 model built in tensorflow.
```

Comment:
Can we add a hyperlink to something like https://www.tensorflow.org/tutorials/image_recognition for the text "inception-v3 model built in tensorflow"?

Reply:
Done.